Modules/Journey/QuestDetailsFrame.lua

```diff
@@ -98,6 +118,8 @@ function _QuestieJourney:DrawQuestDetailsFrame(container, quest)
         local starty = startNpc.spawns[startindex][1][2]
         if (startx ~= -1 or starty ~= -1) then
             local startNPCLocLabel = AceGUI:Create("Label")
+            startx = string.format("%.1f", startx)
+            starty = string.format("%.1f", starty)
```

Comment:
The previous coordinates were showing up with ~10 decimal places.

Reply:
Yeah, I noticed that as well, but never had the time to fix it. Thanks! Though I am a fan of 2 decimal places for coordinates, so this should be `%.2f` shouldn't it?

Reply:
Added as suggestions.